src/screens/Text.js

```diff
@@ -132,6 +132,17 @@ const TextPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="skeleton">
+          <Description>
+            Any Skeleton properties which will be applied to the Text when in
+            skeleton mode. For the Text to be in skeleton mode, use the
```

Comment:
Should we include this "For the [ component ] to be in skeleton mode" sentence on button / heading as well?

Reply:
good point